src/main/java/uk/gov/companieshouse/bankruptofficersearch/api/exception/OracleQueryApiException.java

```diff
@@ -1,11 +1,8 @@
 package uk.gov.companieshouse.bankruptofficersearch.api.exception;
 
-import org.springframework.http.HttpStatus;
-import org.springframework.web.client.HttpClientErrorException;
+public class OracleQueryApiException extends RuntimeException {
```

Comment:
It would be better to use an Exception so it's clearly thrown up to the controller otherwise devs working on the service or controller might not realise they should be catching this one. We can't add context to the logs (IDs, etc) if this is caught by the global exception handler.

Reply:
Changed to extend Exception instead of RuntimeException